src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -64,139 +67,155 @@
   /**
    * Create the GapicProviders based on the given id
    */
-  public static List<GapicProvider<? extends Object>> defaultCreate(Model model,
-      ApiConfig apiConfig, String id) {
+  public static List<GapicProvider<? extends Object>> defaultCreate(
+      Model model, ApiConfig apiConfig, String id) {
 
     // Please keep the following IDs in alphabetical order
 
     if (id.equals(CLIENT_CONFIG)) {
-      GapicProvider<? extends Object> provider = CommonGapicProvider.<Interface>newBuilder()
-          .setModel(model)
-          .setView(new InterfaceView())
-          .setContext(new ClientConfigGapicContext(model, apiConfig))
-          .setSnippetSetRunner(new ClientConfigSnippetSetRunner<Interface>())
-          .setSnippetFileNames(Arrays.asList("json.snip"))
-          .build();
+      GapicProvider<? extends Object> provider =
```

Comment:
I'd prefer the reformatting to be done in a different change - @pongad 's PR https://github.com/googleapis/toolkit/pull/175 will do this. Let's get his submitted before yours and rebase.